services/hanke-service/src/main/resources/db/changelog/changesets/insert-organization-data.yml

```diff
@@ -0,0 +1,158 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-organization-data
+      comment: Inser organization data
+      author: Anne-Mari Silvast
+      changes:
+        - insert:
+            tableName: organisaatio
+            columns:
+              - column: { name: id, value: "1" }
```

Comment:
'id' column entries/values are not needed, as the column is already auto-increment.

Reply:
Ah, good catch, fixed!